cluster/gke/util.sh

```diff
@@ -68,12 +69,28 @@ function test-build-release() {
 
 # Verify needed binaries exist.
 function verify-prereqs() {
-  echo "... in verify-prereqs()" >&2
-
-  ${GCLOUD} preview --help >/dev/null || {
-    echo "Either the GCLOUD environment variable is wrong, or the 'preview' component"
-    echo "is not installed. (Fix with 'gcloud components update preview')"
-  }
+  local cmd
+  for cmd in gcloud; do
```

Comment:
No reason to have a loop here any longer since the logic in the (single) iteration is gcloud specific. If you add another required command the whole loop body would become an if/else anyway.